polaris-react/src/components/AlphaStack/Stack.scss

```diff
@@ -0,0 +1,99 @@
+.Stack {
+  --pc-stack-spacing: var(--p-space-4);
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  margin-top: calc(-1 * var(--pc-stack-spacing));
+  margin-left: calc(-1 * var(--pc-stack-spacing));
+
+  > .Item {
+    margin-top: var(--pc-stack-spacing);
+    margin-left: var(--pc-stack-spacing);
+    max-width: 100%;
+  }
+}
+
+.spacing0 {
```

Comment:
We can remove a lot of css in this file by passing the spacing value from props to a style attribute in the tsx file

i.e. 
```tsx
return <div className={className} style={{'--pc-stack-spacing': spacing}}>{itemMarkup}</div>;
```

And we can use the `gap` property instead of margins now which will let us get rid of the negative margins on parent / positive margins on children, and then remove `.Item` altogether as Alex mentioned